tests/integration-tests/clusters_factory.py

```diff
@@ -199,6 +205,42 @@ def cfn_resources(self):
             self.__cfn_resources = retrieve_cfn_resources(self.cfn_name, self.region)
         return self.__cfn_resources
 
+    @property
+    def head_node_substack_cfn_resources(self):
+        """
+        Return the CloudFormation stack resources for the cluster's head node substack.
+        Resources are retrieved only once and then cached.
+        """
+        if not self.__head_node_substack_cfn_resources:
+            self.__head_node_substack_cfn_resources = retrieve_cfn_resources(
+                self.cfn_resources.get("MasterServerSubstack"), self.region
+            )
+        return self.__head_node_substack_cfn_resources
+
+    @property
+    def compute_fleet_substack_cfn_resources(self):
```

Comment:
nit: this only works with hit substack? shall we make it explicit in the function name or handle the sit case?

Reply:
i ended up not using this, so i'm going to get rid of it